src/wallet/address.ts

```diff
@@ -249,6 +256,13 @@ const generateTronPrivateKey = async (mnemonic: string, i: number) => {
         .derive(i)
         .privateKey?.toString('hex') ?? '';
 };
+const generateQtumPrivateKey = async (testnet: boolean, mnem: string,i:number) => {
+    const network=testnet?QTUM_NETWORK_TESTNET:QTUM_NETWORK_MAINNET;
+    const hdwallet = HDKey.fromMasterSeed(await mnemonicToSeed(mnem), network.bip32);
+    console.log(hdwallet.derive(testnet ? TESTNET_DERIVATION_PATH : BTC_DERIVATION_PATH).toJSON())
```

Comment:
BAsed on this. -https://github.com/satoshilabs/slips/blob/master/slip-0044.md use derivation path m/44'/2301'/0'/0